doc/source/ray-air/preprocessors.rst

```diff
@@ -192,3 +204,108 @@ Custom Preprocessors
     :language: python
     :start-after: __custom_stateful_start__
     :end-before: __custom_stateful_end__
+
+
+Which Preprocessors Should I Use?
+---------------------------------
+
+* If you have categorical data, `encode categories <air-encoding-categorical-data>`_.
+* If you have numerical data, `normalize features <air-scaling-numerical-data>`_.
+* If your model expects a tensor or ``ndarray``, create a tensor using
+  :class:`~ray.data.preprocessors.Concatenator`.
+* If your preprocess isn't supported, `implement a custom preprocessor <air-custom-preprocessors>`_.
+* If want to count the frequency of terms in a document,
+  `create a document-term matrix <air-encoding-documents>`_.
+* If you need to apply more than one preprocessor, compose them together with
+  :class:`~ray.data.preprocessors.Chain`.
+* If your data contains missing values, impute them with
+  :class:`~ray.data.preprocessors.SimpleImputer`.
```

Comment:
I feel like this section is a mess. Could use suggestions on how to organize

Reply:
How is this section a mess? Can you elaborate why this subsection is mess?

1. Does it not transition from the previous section?
2. Do you feel we need more text and explanation or scenario to explain why a specific preprocessor should be used?

Reply:
Hmm, it doesn't read grammatically correctly. 

How about -- 
```rst
What preprocessor should I choose?
------------------------------------

The type of preprocessor that you should use depends on the type of data that you have. Below we provide some tips for handling common data formats.

I have Categorical Data Features.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

If you have categorical data, you may want to convert your data into a vector representation. You can use {A, B, C}  to encode your categorical data into a vector representation. The choice of A, B, or C will depend on the type of data you have.

If your data is a label format, use A to encode your categorical data into a vector representation. Let's take a look at an example:
...
If your data has ordered categories, use B to encode your categorical data into a vector representation. Let's take a look at an example:
...

I have numerical data (floats, ints)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.....
```